### PR TITLE
Remove the OpenGL-based compositor.

### PR DESCRIPTION
**Pull request by jerzywilczek**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -21,7 +21,6 @@
         "parseutils.h": "c"
     },
     "rust-analyzer.linkedProjects": [
-        "native/membrane_videocompositor_opengl_rust/Cargo.toml",
         "native/membrane_videocompositor_wgpu/Cargo.toml"
```
Comment (DominikWolek) [inline .vscode/settings.json:24]: Maybe we should rename the forlder to the `native/membrane_video_compositor/`

**Review by WojciechBarczynski — CHANGES_REQUESTED** [review]
Remove OpenGL in demos and benchmarks as well

**Review by WojciechBarczynski — APPROVED** [review]
lgtm

File: benchmark/lib/benchee/h264_pipeline.exs [hunk 2]
```diff
@@ -61,10 +61,9 @@ defmodule Membrane.VideoCompositor.Benchmark.Benchee.H264 do
 
     Benchee.run(
       %{
-        # FIXME: Add wgpu when it's ready
-        # FIXME: Fixes like the above shouldn't be manual
-        "OpenGL Rust - Two videos into one h264 pipeline benchmark" => fn options ->
-          run_h264_pipeline(%{options | implementation: :opengl_rust})
+        # FIXME: inputting the implementations manually is a really bad design and I hate it ~jerzy
```
Comment (Janix4000) [inline benchmark/lib/benchee/h264_pipeline.exs:64]: +1

File: benchmark/lib/benchee/raw_pipeline.exs [hunk 3]
```diff
@@ -62,10 +62,10 @@ defmodule Membrane.VideoCompositor.Benchmark.Benchee.Raw do
 
     Benchee.run(
       %{
-        # FIXME: Add wgpu when it's ready.
-        # FIXME: Fixes like the above shouldn't be manual
-        "OpenGL Rust - Two videos into one raw pipeline benchmark" => fn options ->
-          run_raw_pipeline(%{options | implementation: :opengl_rust})
+        # FIXME: inputting the implementations manually is a really bad design and I hate it ~jerzy
```
Comment (Janix4000) [inline benchmark/lib/benchee/raw_pipeline.exs:65]: +1

File: lib/membrane/video_compositor/common.ex [hunk 4]
```diff
@@ -28,7 +28,7 @@ defmodule Membrane.VideoCompositor.Implementations.Common.RawVideo do
   end
 end
 
-defmodule Membrane.VideoCompositor.Implementations.Common.Position do
+defmodule Membrane.VideoCompositor.Common.Position do
   @moduledoc """
   A Position struct describing the video position for use with the rust-based compositor implementation.
   Position relative to the top right corner of the viewport, in pixels.
```
Comment (Janix4000) [inline lib/membrane/video_compositor/common.ex:34]: Remember `right` -> `left`

**Review by Janix4000 — APPROVED** [review]

**Pull request merged**